docs/modules/deployment/pages/core/centos-rhel/initialize-core.adoc

```diff
@@ -40,6 +40,16 @@ sudo /opt/opennms/bin/runjava -s
 sudo /opt/opennms/bin/install -dis
 ----
 
+.Identify the OpenNMS group id and allow ICMP (ping)
+[source, console]
+----
+id -g opennms
+# 1000 <1>
+sudo sysctl -w net.ipv4.ping_group_range=1000 1000 >> /etc/sysctl.d/99-opennms.conf <2>
```

Comment:
since there's a space, the range needs to be quoted:

`sudo sysctl -w net.ipv4.ping_group_range="1000 1000"`

Reply:
Also I don't know if you care to make a note that the debian/rpm packages will do this for you automatically

Reply:
> Also I don't know if you care to make a note that the debian/rpm packages will do this for you automatically

That would be awesome!

Reply:
It already does, that's why I said.  :D

(Still worth leaving it in for manual installs...)